src/backend/base/langflow/logging/logger.py

```diff
@@ -136,12 +140,30 @@ class LogConfig(TypedDict):
     log_env: NotRequired[str]
 
 
+class AsyncFileSink(AsyncSink):
+    def __init__(self, file):
+        self._sink = FileSink(
+            path=file,
+            rotation="10 MB",  # Log rotation based on file size
+        )
+        super().__init__(self.write_async, None, ErrorInterceptor(_defaults.LOGURU_CATCH, -1))
```

Comment:
Unfortunately, we can't get the handle id here. But I don't think this is a big problem.

Reply:
I agree, unless we can define it in the `logger.configure(` call